api/ceramic.yaml

```diff
@@ -106,6 +106,12 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Event'
+        '400':
+          description: Event not found
```

Comment:
nice! maybe change this to 404?

In http status codes, 400 means BadRequest and 404 is not NotFound. Calling this 400 EventNotFound is a bit misleading, but the value returned to clients is what they will use, not the internal description. We could have 400 BadRequest, 404 NotFound, etc but I'm not sure what our expected status code support is. 

I was thinking we could just return application/json in all cases but text/plain is probably fine. Even if the client passes an Accept header, we can return whatever we want. Going to approve, but I do think this change is worth making.

Reply:
Agreed. We also have a message that captures the problem "Event not Found"